interpolator/interpolator.F90

```diff
@@ -1173,11 +1164,6 @@ subroutine interpolator_init( clim_type, file_name, lonb_mod, latb_mod, &
 
 module_is_initialized = .true.
 
-!---------------------------------------------------------------------
-!    write version number and namelist to logfile.
-!---------------------------------------------------------------------
-call write_version_number("INTERPOLATOR_MOD", version)
-
```

Comment:
I'm not sure a decision has been made to remove this.

Reply:
Ok. I'll put it back.

Reply:
Was it decided to remove this?  Now I don't remember, and it doesn't look like you put it back...

Reply:
I don't think so. 

I put it back in https://github.com/NOAA-GFDL/FMS/pull/419/commits/bf63e86e60a3457563367446234ebcdc8b45ffbb